Procfile

```diff
@@ -0,0 +1,2 @@
+web: ./docker/run-prod.sh
```

Comment:
Have we tested adding a `cmd` proc type to a Procfile? That `cmd` type is added automatically by deis for a dockerfile deploy.